src/WebUI/src/assets/themes/oruga-tailwind/components/dropdown.css

```diff
@@ -28,7 +24,7 @@
 
 /*  */
 .v-popper--theme-tooltip .v-popper__inner {
-  @apply max-w-xs rounded-lg bg-base-200 px-4 py-3.5 text-content-100;
+  @apply max-w-xs rounded-lg bg-base-200 px-4 py-3.5 text-content-100 drop-shadow-md;
```

Comment:
Added a shadow to give contrast in this scenario:
<details>
  <summary>bg of the hover row of the table matches bg of the tooltip</summary>
  
  ![image](https://github.com/verdie-g/crpg/assets/33551334/c89acae1-0427-495b-b79a-4521fecaf6b5)
</details>